sdks/python/gen_protos.py

```diff
@@ -76,6 +77,14 @@ def generate_proto_files(force=False):
     try:
       from grpc_tools import protoc
     except ImportError:
+      if platform.system() == 'Windows':
+        # For Windows, grpcio-tools has to be installed manually.
+        logging.warning(
```

Comment:
<!--thread_id:cc_145496523_t; commit:0ca6d3025c0479d7e6bd3a70bca84f651e717167; resolved:1-->
<!--section:context-quote-->
> **robertwb** wrote:
> Raise this as an exception. Otherwise the build will "succeed" and they'll get much harder to diagnose errors later.

<!--section:body-->
Done.